MPF.Frontend/Tools/SubmissionGenerator.cs

```diff
@@ -417,7 +449,22 @@ private static SubmissionInfo CreateDefaultSubmissionInfo(BaseProcessor processo
                 return driveLabel;
             }
 
-            // If only one label, don't mention fs
+            // Remove duplicate/useless volume labels
+            string? defaultLabel = SimplifyVolumeLabel(driveLabel);
+            if (defaultLabel != null && defaultLabel.Length != 0)
+            {
+                List<string> keysToRemove = new List<string>();
```

Comment:
Would you consider using something like a `Dictionary<string, List<string>>` and create a reverse map of normalized labels to their corresponding filesystems? That may allow you to do the precedence matching that you proposed before. This would also allow for key removal as you go.

Reply:
Something like that may be best to make this fully featured, but I'm just looking to make this sufficient for now.

Reply:
Please leave a quick TODO in there for this future update and I'll call this gribble set.